src/app/pages/auth/login/LoginView.js

```diff
@@ -14,13 +16,28 @@ LoginView.defaultProps = {
   valid: true,
 }
 
+// ussage with variables
+const title = gettext('Login')
+
 export default function LoginView({ handleSubmit, submitting, valid }) {
+  useEffect(() => {
+    notification.success({
+      description: gettext('message description'),
+    })
+  }, [])
+
   return (
     <form onSubmit={handleSubmit} className={classNames.login}>
-      <h2>Login</h2>
-      <TextField name="email" label="Email" type="email" />
-      <TextField name="password" label="Password" type="password" />
-      <button disabled={submitting || !valid }>Login</button>
+      <h2>{title}</h2>
+      <TextField name="email" label={gettext('Email')} type="email" />
+      <TextField name="password" label={gettext('Password')} type="password" />
+      <button disabled={submitting || !valid }>{gettext('Login')}</button>
+      <span dangerouslySetInnerHTML={{
+        __html: interpolate(
```

Comment:
sometimes we need to add html into `interpolate`. we need to suggest another solution  for such cases

Reply:
const { gettext } = useTranslations()

Reply:
we have HOC + HOOK + render prop + global function
https://django-stars.github.io/frontend-docs/docs/i18n/i18n_about#withtranslations-hoc
https://django-stars.github.io/frontend-docs/docs/i18n/i18n_about#usetranslations-hook
https://django-stars.github.io/frontend-docs/docs/i18n/i18n_about#translator-render-props

Reply:
same example as dangerouslySetInnerHTML we can use for placeholder, alert, promt...